### PR TITLE
move documentation from wiki to github

### PR DESCRIPTION
**Pull request by KasperHeyndrickx**

### CODE REVIEW

**Comment by casz** [comment]
link to see current commit author: https://github.com/jenkinsci/msbuild-plugin/commit/a7a4862f2feee787a4fd4a28f830ba6e332e0277.patch

@KasperHeyndrickx 
You might want to set up an email alias on your GitHub account 🥇
https://help.github.com/articles/adding-an-email-address-to-your-github-account/

Or look into fixing your git user.email config 😅  
https://help.github.com/articles/setting-your-commit-email-address-in-git
then you would need to amend your commits: https://stackoverflow.com/a/3042512
after changing author

to get proper credit for your commits 🏆

**Comment by KasperHeyndrickx** [comment]
@marshall777 Could you take a look at these changes? 
This moves documentation from wiki to git. https://jenkins.io/blog/2019/10/21/plugin-docs-on-github/

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+[![Build Status](https://ci.jenkins.io/job/Plugins/job/msbuild-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/msbuild-plugin/job/master/)
+
+This plugin allows you to use MSBuild to build .NET and Visual Studio
+projects.
+
+## Usage
+
+To use this plugin, specify the location directory of MSBuild.exe on
+Jenkin's configuration page. The MSBuild executable is usually
+situated in a subfolder of C:\\WINDOWS\\Microsoft.NET\\Framework. The
```
Comment (casz) [inline README.md]:
```suggestion
situated in a subfolder of `C:\\WINDOWS\\Microsoft.NET\\Framework`. The
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Visual Studio 2017 Community version of "msbuild.exe" is located in
+"C:\\Program Files (x86)\\Microsoft Visual
+Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\" If you have multiple
```
Comment (casz) [inline README.md]: `"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\"`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+MSBuild versions installed, you can configure multiple executables. 
+
+
```
Comment (casz) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+![](docs/images/jenkins-msbuild.png)
+
+Then, on your project configuration page, specify the name of the build
+file (.proj or .sln) and any [command line
+arguments](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-command-line-reference?view=vs-2017){.external-link}
+you want to pass in. The files are compiled to the directory where
+Visual Studio would put them as well.
+
+
```
Comment (casz) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+"C:\\Program Files (x86)\\Microsoft Visual
+Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\" If you have multiple
+MSBuild versions installed, you can configure multiple executables. 
+
+  
+
+![](docs/images/jenkins-msbuild.png)
+
+Then, on your project configuration page, specify the name of the build
+file (.proj or .sln) and any [command line
```
Comment (casz) [inline README.md]:
```suggestion
file (`.proj` or `.sln`) and any [command line
```

File: README.md (hunk 2 above)
Comment (KasperHeyndrickx) [reply README.md]: Not sure what you're suggesting here?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+This plugin allows you to use MSBuild to build .NET and Visual Studio
+projects.
+
+## Usage
+
+To use this plugin, specify the location directory of MSBuild.exe on
+Jenkin's configuration page. The MSBuild executable is usually
+situated in a subfolder of `C:\\WINDOWS\\Microsoft.NET\\Framework`. The
+Visual Studio 2017 Community version of "msbuild.exe" is located in
+"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\" If you have multiple
```
Comment (casz) [inline README.md]:
```suggestion
`"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\"` If you have multiple
```

File: README.md (hunk 2 above)
Comment (casz) [reply README.md]: I was trying to suggest https://github.com/jenkinsci/msbuild-plugin/pull/39#discussion_r353014408 😅

**Pull request merged**